### DISCUSSION TITLE
Reinitialize or destroy select (Livewire)

### DISCUSSION THREAD

**Discussion by andreasmatuszewski** [orchidjs/tom-select#268] [Q&A] [answered]
Is there a way to reinitialize or destroy the select like in Select2 ($('#mySelect2').select2('destroy');)?

I want to use it for a select field within a modal in a Livewire Component. 

Has someone any ideas?

**Comment by oyejorge** [comment, marked as answer]
https://tom-select.js.org/docs/api/#destroy

**Reply by andreasmatuszewski** [reply]
Thank you for your quick reply!

But sadly i can't call the method like in this example (https://jsfiddle.net/jx7u4rgp/7/). Error: Uncaught TypeError: Cannot read properties of null (reading 'tomselect'). I would try sync(), because i selected the correct options.

**Reply by ToshY** [reply]
@andreasmatu In the fiddle you provided you do the following:

```js
var select = document.getElementById('#select-tags');
```

which should be

```js
var select = document.getElementById('select-tags');
```

**Reply by andreasmatuszewski** [reply]
@ToshY Oh my god, thanks. I should have taken a nap..
